op-proposer/bindings/l2outputoracle.go

```diff
@@ -26,8 +26,16 @@ var (
 	_ = common.Big1
 	_ = types.BloomLookup
 	_ = event.NewSubscription
+	_ = abi.ConvertType
 )
 
+// TypesEOTSInfo is an auto generated low-level Go binding around an user-defined struct.
+type TypesEOTSInfo struct {
```

Comment:
also need proof of inclusion

Reply:
Noob question: where is the Solidity source code for this type?

Reply:
It is added by this [PR](https://github.com/babylonchain/optimism/pull/4) in the op-contracts `Types.sol`(under `packages/contracts-bedrock/src/libraries/`)